libarchive/PKGBUILD

```diff
@@ -24,46 +24,34 @@ prepare() {
   # msysize patch
   #patch -Np1 -i "${srcdir}/libarchive-3.3.1-msys2.patch"
   patch -Np1 -i "${srcdir}/libarchive-3.3.2-bcrypt-fix.patch"
-  autoreconf -ivf
 }
 
 build() {
-  export lt_cv_deplibs_check_method='pass_all'
-  cd "${pkgname}-${pkgver}"
+  local build_type=Release
+  if check_option "debug" "y"; then
+    build_type=Debug
+  fi
+
+  #export lt_cv_deplibs_check_method='pass_all'
 
-  ./configure \
-    --prefix=/usr \
-    --enable-shared \
-    --enable-static \
-    --without-libiconv-prefix \
-    --without-xml2 \
-    --without-cng \
-    --without-lzo2
+  mkdir -p "${pkgname}-${pkgver}/build-${CHOST}"
+  cd "${pkgname}-${pkgver}/build-${CHOST}"
 
-# CNG breaks pacman on Vista due to BCryptDeriveKeyPBKDF2
+  cmake \
+    -G"Unix Makefiles" \
```

Comment:
Have you tried using Ninja?

Reply:
I didn't since I wasn't sure `DESTDIR` worked with Ninja. Apparently it works. However this issue with Ninja is annoying: https://github.com/ninja-build/ninja/issues/1482

Reply:
Yes it is.  `CMAKE_BUILD_PARALLEL_LEVEL` can be used with `cmake --build` (instead of calling ninja directly)

Reply:
Hmm I should amend the commit with that. Thanks for the info @jeremyd2019